src/GenFu/Fillers/DateTimeNullableFillers.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+namespace GenFu
+{
+    public class DateTimeNullableFiller : PropertyFiller<DateTime?>
```

Comment:
Seems like most of this class could delegate to the existing DateTimeFiller with only a small special case to handle the probabilistic filling in of nullables.